src/blocks/advancedheading/ai-text/ai-text.js

```diff
@@ -81,6 +82,7 @@ export const AIText = {
 		const activeStorage = SafeParseJSON( localStorage.getItem( 'kadenceBlocksPrebuilt' ), true );
 		const savedCredits = ( undefined !== activeStorage?.credits && '' !== activeStorage?.credits && null !== activeStorage?.credits ? activeStorage.credits : 'fetch' );
 		const currentCredits = ( '' !== credits ? credits : savedCredits );
+		const currentPostId = parseInt((new URLSearchParams(window.location.search)).get('post')) ?? 0;
```

Comment:
This won't work in widget areas, customizer, FSE, lets just not worry about capturing that in the events as we don't need it